components/notification/NotificationPreferences.client.vue

```diff
@@ -143,6 +143,7 @@ onActivated(() => (busy = false))
               </fieldset>
               <div flex="~ col" gap-y-4 gap-x-2 py-1 sm="~ justify-between flex-row">
                 <button
+                  type="submit"
```

Comment:
there is no need to add it, it is the default

Reply:
According @antfu suggestion, I drop it commit.